project/src/store/user-process/selectors.ts

```diff
@@ -5,3 +5,11 @@ export const getAuthorizationStatus = (state: State): AuthorizationStatus =>
   state[NameSpace.User].authorizationStatus;
 export const getAuthCheckedStatus = (state: State): boolean =>
   state[NameSpace.User].authorizationStatus !== AuthorizationStatus.Unknown;
+
+export const getAuthLogInStatus = (state: State): boolean =>
+  state[NameSpace.User].authorizationStatus === AuthorizationStatus.Auth;
```

Comment:
В данном случае селектор `getAuthLogInStatus` не только возвращает значение, но и выполняет проверку статуса. Поскольку в нашем проекте принято начинать названия селекторов с `get`, предлагаю использовать более описательное название, чтобы подчеркнуть, что селектор выполняет не только выборку, но и логическую проверку. Например, `getIsUserAuthenticated` будет отражать суть селектора лучше. Как ты думаешь?



```suggestion
export const getIsUserAuthenticated = (state: State): boolean =>
  state[NameSpace.User].authorizationStatus === AuthorizationStatus.Auth;
```